cmd/main_test.go

```diff
@@ -1,88 +0,0 @@
-//go:build unittests
-// +build unittests
-
-package main
-
-import (
-	"fmt"
-	"os"
-
-	"github.com/redhat-cne/cloud-event-proxy/pkg/common"
-	"github.com/redhat-cne/sdk-go/pkg/types"
-
-	"sync"
-	"testing"
-
-	"github.com/redhat-cne/sdk-go/pkg/channel"
-	log "github.com/sirupsen/logrus"
-	"github.com/stretchr/testify/assert"
-
-	v1pubsub "github.com/redhat-cne/sdk-go/v1/pubsub"
-)
-
-var (
-	resourceAddress string = "/test/main"
-)
-
-func storeCleanUp() {
-	_ = scConfig.PubSubAPI.DeleteAllPublishers()
-	_ = scConfig.PubSubAPI.DeleteAllSubscriptions()
-}
-
-func TestSidecar_MainWithHTTP(t *testing.T) {
```

Comment:
do not delete the test file. the TC can be updated to test start up without HTTP.